packages/rollup/test/integration/BUILD.bazel

```diff
@@ -0,0 +1,44 @@
+load("@build_bazel_rules_nodejs//internal/golden_file_test:golden_file_test.bzl", "golden_file_test")
```

Comment:
could you move this integration test to a separate PR? a lot of extra code and we should discuss why have both this and the existing integration test

Reply:
existing integration test was using the goldens from the old rollup_bundle to make sure that the outputs were identical; I expanded on that but wanted to keep the existing one as an assertion until the old rollup_bundle is removed

Reply:
I'll factor it out to a separate PR